jetty-osgi/jetty-osgi-boot/src/main/java/org/eclipse/jetty/osgi/boot/internal/webapp/OSGiWebappClassLoader.java

```diff
@@ -274,35 +266,4 @@ private boolean isAcceptableLibrary(File file, Set<String> pathToClassFiles)
     }
 
     private static Field _contextField;
-
```

Comment:
It's been a while. I don't recall.
Obviously nothing is using it in our code. (no references to it)

The setAccessible possibly?  but that can be easily fixed with method/field modifiers.

But I bet it's more likely to be related to the `addClassPath(webappContext.getExtraClasspath())` which is additive to existing classpath from prior context (not a replacement).

Reply:
The WebAppClassLoader._context is a final field.
You cannot reset the value.

This method is broken in many different ways.
Nobody is using it, otherwise we would have had many bug reports against it already.

Reply:
I had to touch it as `webappContext.getExtraClasspath()` returns a `List<Resource>` now.